plugin/neogit.vim

```diff
@@ -1,19 +1,19 @@
 lua require("neogit")
 lua require("neogit.status")
 
-function! s:refresh()
+function! s:refresh(file)
   if match(bufname(), "^\\(Neogit.*\\|.git/COMMIT_EDITMSG\\)$") == 0
     return
   endif
-  lua vim.defer_fn(function() require 'neogit.status'.refresh() end, 0)
+  call luaeval('(function() require "neogit.status".dispatch_refresh({ status = true, diffs = {_A}}) end)()', a:file)
```

Comment:
Is there a reason why you changed to `luaeval`?

Reply:
Yeah I needed to pass in the filename from viml. Is there a way to do that with `:lua`?

Reply:
Hmm.... it doesn't work properly anyway, we have the usual problem when `cwd` is not in the repository root, becase the filename doesn't line up with git's representation.